run_test.sh

```diff
@@ -51,7 +51,7 @@ echo "cd $PWD"
 git clone --branch development https://github.com/AMReX-Codes/amrex.git
 # Use QED brach for QED tests
 if [ "${WARPX_CI_QED}" = "TRUE" ]; then
-    git clone --branch QED https://github.com/ECP-WarpX/picsar.git
+    git clone --branch improved https://github.com/lucafedeli88/picsar.git
```

Comment:
Please don't forget to update to:
```suggestion
    git clone --branch development https://github.com/ECP-WarpX/picsar.git
```
when merging :)

Reply:
Sure!